apps/obsidian/src/components/TldrawView.tsx

```diff
@@ -106,7 +106,11 @@ export class TldrawView extends TextFileView {
         return;
       }
 
-      const { store } = processInitialData(data, assetStore);
+      const { store } = processInitialData(data, assetStore, {
+        app: this.app,
+        canvasFile: this.file!,
```

Comment:
Is there a specific reason we need to use `!` here?  `!` is generally unsafe and should be avoided whenever possible and only used as a last resort.